### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by mpa-mxiang**
For this PR:

- Understand the concept of state management.
- Use store, actions and reducers in React.
- Connect React and Redux.
- Apply React best practices and language style guides in code.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -23,10 +34,54 @@ h1 {
...
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.CATEGORIES {
+  width: 6.563rem;
+  height: 1rem;
+  margin: 1.125rem 41.5rem 0.688rem 2.563rem;
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
```
Comment (rloterh) [inline src/App.css]: In this activity is styling, we want the design of your app to closely mimic the [design template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) on Zeplin. Please bear in mind that the CSS code for this design can be easily obtained by simply clicking on a particular element of field and the CSS will be displayed on the right side, as seen in the attached screenshot below. Let's ensure to introduce the following improvements to our design:
- [x] Ensure that all input fields, together with the add book button within the form area are on one horizontal line, as seen in the Zeplin template.
- [x] Let's improve upon the typography by using the correct font face, size, and color
- [x] Ensure to add the user icon at the top right
- [x] Apply correct background styling
- [x] Elements should be positioned correctly, and appropriate spacing added as seen in the Zeplin template.

**Your design:**
image

**Expected (You can find the CSS for a particular object and CSS properties by clicking on that object in Zeplin):**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Jiachun, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁 🚦 🏃‍♀️ 🌴 🏫 🌳 🏘️

## Highlights 👍  
- Good job with the design so far!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @mpa-mxiang  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [x]  There should be not that big right margin. KIndly adjust according to original website 
- [x] There should be a separate line after percentage, marked in picture, kindly add it.
- [ ] [OPTIONAL] there is ` comments, remove & edit ` marked in the original website this should be present with a given small margin. KIndly add it.
- Your website

Screenshot from 2023-06-17 23-21-07
- Original website

Screenshot from 2022-11-19 15-37-13

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @mpa-mxiang 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Descriptive PR.
- Good commit messages.
- Correct git-flow.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mpa-mxiang** [comment]
Thank you for the approval! @emmanuellekamwa

**Pull request merged**